specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
As this change is in stable API version, please get a breaking changes review done from Breaking Changes Review Board <azbreakchangereview@microsoft.com>.

Reply:
The change is not released. However I will drop an email to the board and get back,